crates/uv-virtualenv/src/activator/activate_this.py

```diff
@@ -44,13 +45,13 @@
 # prepend bin to PATH (this file is inside the bin directory)
 os.environ["PATH"] = os.pathsep.join([bin_dir, *os.environ.get("PATH", "").split(os.pathsep)])
 os.environ["VIRTUAL_ENV"] = base  # virtual env is right above bin directory
-os.environ["VIRTUAL_ENV_PROMPT"] = "" or os.path.basename(base)  # noqa: SIM222
+os.environ["VIRTUAL_ENV_PROMPT"] = "{{ VIRTUAL_PROMPT }}" or os.path.basename(base)  # noqa: SIM222
 
 # add the virtual environments libraries to the host python import mechanism
 prev_length = len(sys.path)
 for lib in "{{ RELATIVE_SITE_PACKAGES }}".split(os.pathsep):
     path = os.path.realpath(os.path.join(bin_dir, lib))
-    site.addsitedir(path.decode("utf-8") if "" else path)
+    site.addsitedir(path)
 sys.path[:] = sys.path[prev_length:] + sys.path[0:prev_length]
```

Comment:
(Ignorable nit, but since we're here)

```suggestion
sys.path[:] = sys.path[prev_length:] + sys.path[:prev_length]
```

Reply:
Thanks, although unlikely to change this since every deviation makes it a bit more difficult to keep these in-sync going forward.